api/v1alpha1/siteconfig_types.go

```diff
@@ -329,8 +329,12 @@ type ManifestReference struct {
 	APIGroup *string `json:"apiGroup,omitempty"`
 	// Kind is the type of resource being referenced
 	Kind string `json:"kind,omitempty"`
-	// Name is the name of resource being referenced
+	// Name is the name of the resource being referenced
 	Name string `json:"name,omitempty"`
+	// Namespace is the namespace of the resource being referenced
+	Namespace string `json:"namespace,omitempty"`
```

Comment:
Not sure this belongs in the sync wave PR.

Reply:
The namespace is required for specifying the object to be deleted in the finalizer section.

Reply:
The namespace here serves a different purpose. It is used for specifying the namespace of the rendered manifests whereas #7 is for specifying extra-manifests outside of the local namespace. Furthermore, an empty namespace allows us to identify that this is a cluster scoped resource (such as ManagedCluster kind).

Reply:
Ah okay, so will this only ever be the namespace of the SiteConfig or empty?